File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/01_First_Steps_In_Coding_Exercise/07_Food_Delivery.js
function foodDelivery(input) {
    let chickenPrice = 10.35;
    let fishPrice = 12.40;
    let vegPrice = 8.15;

    let delivery = 2.50;

    let chickenMenus = Number(input[0]);
    let fishMenus = Number(input[1]);
    let vegMenus = Number(input[2]);

    let totalPrice = chickenMenus * chickenPrice + fishMenus * fishPrice + vegMenus * vegPrice;

    let dessert = totalPrice * 0.20;

    let finalTotalPrice = totalPrice + dessert + delivery;

    console.log(finalTotalPrice);
}


// foodDelivery(["2", "4", "3"]);

File: Python_Courses/Python_Basics/Preparation/01_First_Steps_In_Coding_Exercise/07_Food_Delivery.py
CHICKEN_PRICE = 10.35
FISH_PRICE = 12.40
VEG_PRICE = 8.15
DELIVERY_PRICE = 2.50

chicken_count = int(input())
fish_count = int(input())
veg_count = int(input())

total_price_without_dessert = chicken_count * CHICKEN_PRICE + fish_count * FISH_PRICE + veg_count * VEG_PRICE
dessert_price = total_price_without_dessert * 0.20

total_price = total_price_without_dessert + dessert_price + DELIVERY_PRICE

print(total_price)
